setup.py

```diff
@@ -45,4 +45,11 @@ def get_install_requirements(path):
       extras_require={
           'avro': ['fastavro', 'requests', avro],
           'dev': get_install_requirements("test-requirements.txt")
+      },
+      entry_points={
```

Comment:
As  mentioned above this should be the responsibility of the application as opposed to the `confluent-kafka-python` project.